README.rst

```diff
@@ -86,18 +97,27 @@ Xcode "build post-action" (Recommended)
 #.  click symbol ``+`` on the left bottom corner of the right pane
 #.  choose ``New Run Script Action``
 #.  choose your selected scheme name in ``Provide build settings from``
-#.  input commands below:
+#.  if you used ``pip`` or direct installation, input commands below:
     
     .. code-block:: bash
 
       $ xunique "${PROJECT_FILE_PATH}/project.pbxproj"
 
-#.  click ``Close`` and it's all done.
+#.  if you used CocoaPods to install, input the following instead:
+
+.. code-block:: bash
+   
+   "${SRCROOT}/Pods/xUnique/xUnique_Run_Script.sh"
+
+#.  click ``Close`` to save your changes.
 #.  Next time when you Build or Run the project, xUnique would be
     triggered after build success. If the build works, you could commit
     all files.
 #. Demo gif animation is `here <https://github.com/truebit/xUnique#add-xunique-to-xcode-post-action>`__
 
+**NOTE:** Do not try to add this as a Run Script Build Action to your actual target, 
+or Xcode will error out since the identifiers have changed since the build began. 
```

Comment:
maybe change "Xcode will error out since the identifiers have changed" to something like "the build will fail at the Run Script phase because the project file has changed"?